google/cloud/bigquery/client.py

```diff
@@ -3307,6 +3318,9 @@ def _list_rows_from_query_results(
             "location": location,
         }
 
+        if timeout is not None:
+            timeout = max(timeout, _MIN_GET_QUERY_RESULTS_TIMEOUT)
```

Comment:
Should we document this in the methods?  e.g. timeout less than the _MIN_GET_QUERY_RESULTS_TIMEOUT are ignored?